aiopogo/auth_ptc.py

```diff
@@ -39,31 +39,55 @@ def __init__(self, username=None, password=None, proxy=None,
 
         try:
             now = time()
+            class CustomClientRequest(ClientRequest):
```

Comment:
I would prefer for these classes to stand alone in this file rather than embedded in this method. This way isn't technically wrong but nested classes are uncommon and potentially confusing to some.